bridgetown-core/lib/bridgetown-core/configurations/stimulus.rb

```diff
@@ -36,7 +36,7 @@
       Object.entries(controllers).forEach(([filename, controller]) => {
         if (filename.includes("_controller.") || filename.includes("-controller.")) {
           const identifier = filename.replace("./controllers/", "")
-            .replace(/[_\-]controller\..*$/, "")
+            .replace(/[_-]controller\\..*$/, "")
```

Comment:
Similarly, these changes appear to change the output of this configuration as well.

```diff
- .replace(/[_-]controller..*$/, "")
+  .replace(/[_-]controller\..*$/, "")
```

Again, I don't think we want to include these.